package.json

```
@@ -5,6 +5,13 @@
   "main": "dist/lexer.cjs",
   "module": "dist/lexer.js",
   "types": "types/lexer.d.ts",
+  "exports": {
+    ".": {
+      "module" "./dist/lexer.js",
+      "import": "./dist/lexer.js",
+      "require": "./dist/lexer.cjs"
+    }
```

One other thing - when there's only a main export, it's possible to flatten exports down to a single map as sugar. Perhaps unnecessary but I like it :)

```suggestion
    "module" "./dist/lexer.js",
    "import": "./dist/lexer.js",
    "require": "./dist/lexer.cjs"
```